pkg/storage/s3/s3.go

```diff
@@ -1224,7 +1224,12 @@ func (is *ObjectStorage) GetBlobContent(repo string, digest godigest.Digest) ([]
 	return buf.Bytes(), nil
 }
 
-func (is *ObjectStorage) GetReferrers(repo string, digest godigest.Digest, mediaType string,
+func (is *ObjectStorage) GetReferrers(repo string, digest godigest.Digest, artifactType string,
```

Comment:
We should add a task to track implementing this?

Reply:
@peusebiu